crates/wash-cli/src/dev.rs

```diff
@@ -534,6 +536,9 @@ struct ProjectDeps {
     /// Lookup of dependencies by project key, with lookups into the pool
     dependencies: HashMap<ProjectDependencyKey, DependencySpec>,
 
+    /// The component under test
+    component: Option<Component>,
```

Comment:
I don't quite get what this help text means 🤔 Is this the component we're running the dev loop for?
\

Reply:
Ah yes! I will change that to make it more clear

Reply:
Fixed!